ferry.go

```diff
@@ -836,13 +837,21 @@ func (f *Ferry) Progress() *Progress {
 		CurrentState:  f.OverallState.Load().(string),
 		CustomPayload: f.Config.ProgressCallback.Payload,
 		VerifierType:  f.VerifierType,
+		OverallState:  f.OverallState.Load().(string),
 	}
 
+	s.GhostferryVersion = VersionString
```

Comment:
Do we use `s` and not `p` since it used to be `Status`?

Reply:
probably.